lib/github.com/mpllang/mpllib/ParFuncArray.sml

```diff
@@ -109,7 +109,7 @@ struct
                 (* | NONE => raise Fail "ParFuncArray.getVersion found empty slot" *)
               )
         in
-          SOME (#2 ((*valOf*) (Array.sub (data, idx))))
+          SOME (#2 (*valOf*) (Array.sub (data, idx)))
```

Comment:
The parens here aren't unnecessary; the commented-out `valOf` would need them if re-enabled.